src/submission/submission-api.ts

```diff
@@ -25,11 +25,22 @@ export enum FileType {
   SAMPLE = 'sample',
 }
 
-export const FileNameRegex = {
-  [FileType.REGISTRATION]: '^registration.*.tsv',
-  [FileType.DONOR]: '^(donor).*.tsv',
-  [FileType.SPECIMEN]: '^(specimen).*.tsv',
-  [FileType.SAMPLE]: '^(sample).*.tsv',
+const FileNameRegex = {
+  [FileType.REGISTRATION]: '^registration.*\\.tsv',
```

Comment:
any reason we arent setting these as regex immediately?

`[FileType.REGISTRATION]:/^registration.*\.tsv/,`